app/routes/docs.faq/route.mdx

```diff
@@ -102,7 +102,7 @@ To get started, [sign in or sign up](https://gcn.nasa.gov/login) and then select
 
 ### Why are my Kafka client credentials expiring?
 
-GCN has updated our security practices to disable disused Kafka client credentials that have not connected to one of our Kafka brokers within the last month. The [client credentials page](/user/credentials) lists the date that the credential was last used to connect to one of the GCN Kafka brokers. Users will receive an email warning in advance of the expiration date. Connecting that client credential to one of the GCN Kafka brokers after the warning email will update the last used date and postpone the expiration. Users may delete expired client credentials and generate new client credentials at any time in the [client credential page](/user/credentials).
+For security purposes, we will disable client credentials that you have not used to connect to a Kafka broker for 30 the past days. We will email you reminders 9 days before client credentials will expire. Once disabled, a client credential cannot be used again. To prevent a credential from expiring and being disabled, simply use it to connect to a Kafka broker. You may create new client credentials at any time. To review, create, delete, and manage your client credentials, see your account [client credential settings](https://gcn.nasa.gov/user/credentials).
```

Comment:
```suggestion
For security purposes, we will disable client credentials that you have not used to connect to a Kafka broker for the past 30 days. We will email you reminders 9 days before client credentials will expire. Once disabled, a client credential cannot be used again. To prevent a credential from expiring and being disabled, simply use it to connect to a Kafka broker. You may create new client credentials at any time. To review, create, delete, and manage your client credentials, see your account [client credential settings](https://gcn.nasa.gov/user/credentials).
```